src/Raven.Client/Documents/Session/AbstractDocumentQuery.cs

```diff
@@ -932,7 +932,32 @@ public void AndAlso()
 
             tokens.AddLast(QueryOperatorToken.And);
```

Comment:
let's remove code redundancy. We can execute here `AndAlso(wrapPreviousQueryClauses: false)` right?

Reply:
indeed